polling/infrequent/activities.py

```diff
@@ -13,7 +13,8 @@ class ComposeGreetingInput:
 
 @activity.defn
 async def compose_greeting(input: ComposeGreetingInput) -> str:
-    test_service = TestService()
+    attempt = activity.info().attempt - 1
+    test_service = TestService(attempt=attempt)
```

Comment:
Thanks! Seems this was written with a bug.

I think a cleaner approach would be to make this activity a method (and call via `execute_activity_method` from workflow) and instantiate the `TestService` as class state.

Also, I think  `frequent` polling is affected by this same test service issue.